crates/firewheel-core/Cargo.toml

```diff
@@ -17,9 +17,14 @@ all-features = true
 [features]
 symphonium = ["dep:symphonium"]
 resampler = ["symphonium?/resampler"]
+bevy = ["dep:bevy_ecs"]
 
 [dependencies]
+firewheel-macros.workspace = true
```

Comment:
Could this be an optional dependency that only gets included if the "bevy" feature is enabled?

Reply:
Oh wait, nevermind. I got the bevy derive macros mixed up with the firewheel derive macros.